source/index.rst

```diff
@@ -3,12 +3,47 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to open5e
-==================================
+Open5e
+======
 
-*The best open source for the SRD, and the source of the best open content*
+Open5e aims to be the best open-source resource for fifth edition d20 content.
```

Comment:
The term "D&D" is explicitly not part of the OGL per the included license. I'm using d20 because they don't provide an alternative.